internal/codespaces/api/api.go

```diff
@@ -42,13 +43,13 @@ import (
 
 	"github.com/cenkalti/backoff/v4"
 	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghinstance"
+	"github.com/cli/cli/v2/pkg/cmdutil"
 	"github.com/opentracing/opentracing-go"
 )
 
 const (
-	githubServer = "https://github.com"
-	githubAPI    = "https://api.github.com"
-	vscsAPI      = "https://online.visualstudio.com"
+	defaultApiUrl = "https://api.github.com"
```

Comment:
```suggestion
	defaultAPIURL = "https://api.github.com"
```
Small naming nit per Go convention.